core/src/main/scala/org/apache/spark/sql/SnappyBaseParser.scala

```diff
@@ -98,7 +98,9 @@ abstract class SnappyBaseParser(session: SparkSession) extends Parser {
   }
 
   protected final def stringLiteral: Rule1[String] = rule {
-    '\'' ~ capture((noneOf("'") | "''").*) ~ '\'' ~ ws ~> ((s: String) =>
+    (("\"" ~ capture((noneOf("\"") | "\"\"").*) ~ "\"") |
+        ("\'" ~ capture((noneOf("'") | "''").*) ~ "\'")) ~
+        ws ~> ((s: String) =>
```

Comment:
Ok. Will merge after discussion.

Reply:
Since strings can be enclosed in double quotes or single quotes, I made this change to fix a syntax which has string within "". Earlier it was only accepting single quoted string

Reply:
@hemanthmeka That's what I am saying. The convention in Derby and other RDBs is that double quotes are for identifiers (column/table names) and NOT strings. All external tools that generate queries use the same convention. Now there is ambiguity since SnappyBaseParser.quotedIdentifier is also interpreting double quotes and strings are too. I think we should revert this change and instead fix the tests.

Reply:
Ok, we will do so.

Reply:
For info, the jira issue is [SNAP-2307](https://jira.snappydata.io/browse/SNAP-2307)